framework/src/tinyalsa/tinyalsa.c

```diff
@@ -123,6 +123,7 @@ struct pcm {
 	struct ap_buffer_s *nextBuf;
 	unsigned int nextSize;
 	unsigned int nextOffset;
+	unsigned int enqueuedBytes;
```

Comment:
This is a violation of [[coding rule]](https://wiki.tizen.org/Native_Platform_Coding_Idiom_and_Style_Guide#Declaration.2FDefinition).

## Naming
[M12] No mixed-case, Use underscores('_') to separate words in an name.

[Example-Do]
```
#define TEST_VALUE	32
extern struct list_head_hash[TEST_VALUE];

int example_function(void)
{
    return(TEST_VALUE - 1);
}
```
[Example-Don't]
```

#define Test_Value	32
extern struct list_head_hash[Test_Value];

int ExampleFunction(void)
{
    return(Test_Value - 1);
}
```

Reply:
Ok I will create another pull request to correct the variable names.

Reply:
newly created variable must follow this but you don't have to modify previous value because we bring them from tinyalsa repo.

Reply:
If you don't want to follow Tizen coding rule, you should move it to external. If you don't move it to external, you should follow ours.

Reply:
You misunderstanding what I mean
Of course we'll follow coding rule. 
Generally, there are some exception in most of coding convention.
Most of case, well-known naming is one of exception case of coding rule. 
We will follow indent and another coding style, but naming of variable, I don't agree.